src/install.js

```diff
@@ -56,11 +57,19 @@ export const DEPENDENCY_FIELDS = [
  * @param  {String} name - name of the dependency.
  * @return {Observable} - observable sequence of `package.json` objects.
  */
-export function resolveLocal (nodeModules, parentTarget, name) {
+export function resolveLocal (nodeModules, parentTarget, name, version) {
 	const linkname = path.join(nodeModules, parentTarget, 'node_modules', name)
 	const fetch = () => EmptyObservable.create()
 	log(`resolving ${linkname} from node_modules`)
 
+	// support `file:` with symlinks
+	if (version.substr(0, 5) === 'file:') {
```

Comment:
It would be better to use [npm-package-arg](https://github.com/npm/npm-package-arg) module to parse dependencies.

It was here, in [1.x version](https://github.com/alexanderGugel/ied/blob/v1.1.1/lib/resolve.js#L15) but was lost during refactoring